### PR TITLE
run mix format

### PR DESCRIPTION
**Pull request by SimonLab**
run `mix format` on the project

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -76,4 +76,4 @@ config :phoenix, :stacktrace_depth, 20
 config :phoenix, :plug_init_mode, :runtime
 
 # add pre-commit command see #20
-config :pre_commit, commands: ["test", "coveralls"]
+config :pre_commit, commands: ["format", "test", "coveralls"]
```
Comment (nelsonic) [inline config/dev.exs]: thanks for adding this. 👍

File: lib/app/ctx/item.ex [hunk 2]
```diff
@@ -4,7 +4,8 @@ defmodule App.Ctx.Item do
 
   schema "items" do
     field :text, :string
-    field :person_id, :id # an item always belongs to a person
+    # an item always belongs to a person
```
Comment (nelsonic) [inline lib/app/ctx/item.ex]: I'm not a fan of `mix format` not allowing in-line comments ... 😞

**Comment by SimonLab** [comment]
sorry @nelsonic I open this PR a bit too soon, the more up to date is #51

File: lib/app_web/controllers/page_controller.ex [hunk 3]
```diff
@@ -12,25 +12,24 @@ defmodule AppWeb.PageController do
   def index(conn, _params) do
     url_oauth_google = ElixirAuthGoogle.generate_oauth_url(conn)
     changeset = Person.changeset_registration(%Person{}, %{})
-    render(conn, "index.html",
-      [url_oauth_google: url_oauth_google, changeset: changeset])
+    render(conn, "index.html", url_oauth_google: url_oauth_google, changeset: changeset)
```
Comment (nelsonic) [inline lib/app_web/controllers/page_controller.ex]: This line is now 88 characters long ... is there a way of telling `mix format` to

File: config/dev.exs (hunk 1 above)
Comment (SimonLab) [reply config/dev.exs]: In fact I'm not sure adding the format command on pre-commit is that useful as any changes produce by `format` won't be added to the commit I think.
The best way is to configure our editor to run `mix format` when a file is saved

**Pull request closed without merging**